### ISSUE TITLE
Feature Request: JMAP Calendars

### ISSUE THREAD

**Issue by klotzambein** [stalwartlabs/jmap-client#3]
Would implementing the proposed [calendar api](https://jmap.io/spec-calendars.html) be within the scope of this project?

Depending on the amount of effort involved, I might be able to submit a PR.

**Comment by mdecimus** [comment]
Yes, support for all JMAP RFCs is planned. This includes JMAP Calendars, JMAP Contacts, JMAP Blob, JMAP Sieve and others, which will be implemented as soon as these become RFCs. The Calendars specification is probably going to become a standard in 2023, according to Fastmail.

**Comment by WhyNotHugo** [comment]
I can work on this feature, but it only makes sense [for my use case] if https://github.com/stalwartlabs/jmap-client/discussions/15 is acceptable.

**Comment by philip-peterson** [comment]
I create a vibe-coded version of this: https://github.com/stalwartlabs/jmap-client/compare/main...philip-peterson:jmap-client:calendar?expand=1
